lib/csv_to_popolo.rb

```diff
@@ -257,6 +262,16 @@ def chambers
       end
     end
 
+    def posts
+      @_posts ||= @csv.select { |r| r.key? :legislative_membership_type }.uniq { |r| r[:legislative_membership_type] }.map do |r|
+        {
+          id: "#{_idify(r[:legislative_membership_type])}",
```

Comment:
I don't think this value needs to be in a string literal, since `_idify` will return a string anyway?